pandas/core/internals.py

```diff
@@ -2020,6 +2020,14 @@ def re_replacer(s):
 
         return block
 
+    def to_native_types(self, slicer=None, na_rep='nan', quoting=None,
+                        bytes_encoding=None, **kwargs):
```

Comment:
move this to ``Block``, where ``to_native_types`` *already* exists, you just need to incorporate this argument after stringifying (we have another conditional if quoting or not, but and am not sure if you need to do this or not for that one as I think the csv writer handles that).

around line 590.